ports/stm32/mboot/main.c

```diff
@@ -1546,23 +1542,6 @@ void stm32_main(int initial_r0) {
         }
         #endif
 
-        #if USE_USB_POLLING
```

Comment:
I think a WFI is now needed in the case USB_USB_POLLING is false, in order to reduce power consumption (before it would be done in the `mp_hal_delay_ms()` calls).

Reply:
I'd missed the WFI in `mp_hal_delay_ms`. I also hadn't been considering power use...